File: Data Structures & Algorithms/find-duplicate-integer/submission-1.java
class Solution {
    public int findDuplicate(int[] nums) {
     int duplicate = 0;
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] == nums[j]) {
                    duplicate = nums[i];
                    break;
                }
            }
            if (duplicate != 0) break;
        }
        return duplicate;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target)
    {
        Dictionary<int, int> dict = new Dictionary<int, int>();
        for (int i = 0; i < nums.Length; i++) 
        {
            dict[nums[i]] = i;
        }
        
        for (int i = 0; i < nums.Length; i++)
        {
            int x = nums[i];
            int diff = target - x;
            
            if (dict.ContainsKey(diff) && dict[diff] != i)
            {
                return new int [] {i, dict[diff]};
                }
            }
            
            return new int [0];
            }
            }
